CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,115 @@
+- National origin
+- Disability
+- Age (40 years or older)
+- Genetic information
+- Parental status
+- Political affiliation
+- Marital status
+- Uniformed status
+- Membership in a labor organization or union activities
+- Prior equal employment opportunity (EEO) or whistleblower activity, or
+- Any other factor unrelated to your merit
+
+The policy also forbids harassing conduct, which includes unwelcome conduct based on any (or a combination of) protected traits or characteristics. Such conduct may take the form of any of the following: 
+
+- Offensive jokes, comments, objects, or pictures 
+- Questions about a person’s identity (e.g., disability status, gender identity, sexual orientation, national origin, etc.) 
+- Undue attention 
+- Ridicule or mockery 
+- Insults or put-downs 
+- Touching/physical contact 
+- Slurs or epithets 
+- Threats or other forms of intimidation 
+- Physical or sexual assault
+
+## Reporting violations
+
+If you are a CFPB employee, former CFPB employee, or job applicant to CFPB and believe you have been 
+discriminated against or harassed on the basis of race, color, religion, sex (including pregnancy, 
+sexual orientation, transgender status, gender identity or expression, gender non-conformity, or sex 
+stereotyping of any kind), national origin, disability, age (40 years or older), genetic information, 
+parental status, or retaliated against for prior Equal Employment Opportunity (EEO) activity, contact the CFPB’s Office of Civil Rights.
+
+CFPB_EEO@consumerfinance.gov
+
+(202) 435-9EEO
+(855) 233-0362
+TTY: (202) 435-9742
+
+Office of Civil Rights
+Consumer Financial Protection Bureau
+1700 G Street, NW
+Washington, D.C. 20552
+
+For help filing a complaint about discrimination on the basis of marital status, political 
+affiliation, or any other non-merit factor, or for claims of retaliation for [whistleblower activity](https://www.consumerfinance.gov/office-civil-rights/whistleblowers/), contact the [Office of Special Counsel](http://www.osc.gov/) or the [Merit Systems Protection Board](http://www.mspb.gov/).
+
+For help filing a complaint about discrimination on the basis of uniformed status, you may contact 
+the [Veterans’ Employment and Training Service (VETS)](http://www.dol.gov/vets/) at the Department of Labor, the [Merit Systems Protection Board](http://www.mspb.gov/), or the [Office of Special Counsel](https://osc.gov/), depending on the circumstances.
+
+For help filing a complaint about discrimination on the basis of membership in a labor organization, 
+you may contact the [Federal Labor Relations Authority](http://flra.gov/) or your union (if applicable).
+
+### Equal employment opportunity policy
+
+For more information about the CFPB’s equal employment opportunity (EEO) policies and procedures visit https://www.consumerfinance.gov/office-civil-rights/eeo-policy-and-reports/ 
```

Comment:
Make https://www.consumerfinance.gov/office-civil-rights/eeo-policy-and-reports/ a link.